Wizard/Install/Form/AdminStepType.php

```diff
@@ -34,6 +34,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'second_name'     => 'password_again',
                 'invalid_message' => 'The password fields must match.',
             ))
+            ->add('timezone', 'timezone')
```

Comment:
@ghostika Could you please make it a searchable select2 field, just like when editing the profile?